RabbitMq/BatchConsumer.php

```diff
@@ -313,7 +313,7 @@ private function getMessage($deliveryTag)
     private function getMessageChannel($deliveryTag)
     {
         $message = $this->getMessage($deliveryTag);
-        if (!$message) {
+        if (empty($message)) {
```

Comment:
In my opinion using empty should be avoided, and we should enforce a more strict control, even if more verbose.

Reply:
@ecourtial do you have a proposal ?

Reply:
@mihaileu we can simply check for null, because `getMessage` should return only `AMQPMessage|null`.

`$message === null`

Reply:
@andreea-anamaria let's do the change and merge this